homeassistant/components/scsgate/__init__.py

```diff
@@ -20,8 +20,6 @@
 
```

Comment:
May you also delete `ATTR_STATE` ? Since ones used in light and switch are from HA const.

Reply:
Sure!